Controllers/ExercisesController.cs

```diff
@@ -461,15 +467,65 @@ public async Task<IActionResult> ExecuteCode(int id, string solution, string lan
             var jsonRequest = JsonConvert.SerializeObject(request);
             var httpContent = new StringContent(jsonRequest, Encoding.UTF8, "application/json");
 
-            var response = await _httpClient.PostAsync("http://localhost:8000/execute", httpContent);
+            try
+            {
+                var response = await _httpClient.PostAsync("http://localhost:8000/execute", httpContent);
+
+                if (response.IsSuccessStatusCode)
+                {
+                    var jsonResponse = await response.Content.ReadAsStringAsync();
+                    var result = JsonConvert.DeserializeObject<Dictionary<string, object>>(jsonResponse);
+
+                    int passedTests = 0;
+                    int failedTests = 0;
+
+                    /*
+                    foreach (var test in result)
```

Comment:
Score field is always 0, no matter the results of the tests. Probably you should fix this function in order to calculate the score correctly.

Reply:
I've fixed the issue, thank you!